source/_docs/guides/drupal-8-simplesamlphp.md

```diff
@@ -0,0 +1,264 @@
+---
+title: How to Implement SimpleSAMLphp as idP for Drupal 8 on Pantheon 
+description: Setting up an identity provider on Drupal 8 using SimpleSAMLphp.
+draft: true
+tags: [code, drupal-8]
+categories: [drupal, developing]
+contributors: Javi-er
+---
+
+There are a number of articles detailing how to implement the SAML 2.0 protocol, but if you want to host your site on Pantheon and use Drupal 8 as your identity provider, there are a few details to take in account.
+<div class="alert alert-info">
+<h4 class="info">Note</h4><p markdown="1">The following tutorial assumes that you are on a linux based computer or a Mac. Also that your project is running on Apache.
```

Comment:
`Also that your project is running on Apache,`

Pantheon uses Nginx, so Apache configurations won't apply.

Reply:
Thanks, I removed any Apache reference except by an example. The Apache requirement was for the developer computer and not the server.